<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>321</width>
      <height>551</height>
     </rect>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Test Mode</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>301</width>
        <height>291</height>
       </rect>
      </property>
      <property name="title">
       <string>Manual Control</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>271</width>
         <height>251</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>VOUT A</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sld_voltA">
            <property name="maximum">
             <number>4095</number>
            </property>
            <property name="pageStep">
             <number>10</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spb_voltA">
            <property name="maximum">
             <number>4095</number>
            </property>
            <property name="displayIntegerBase">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>VOUT B</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sld_voltB">
            <property name="maximum">
             <number>4095</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spb_voltB">
            <property name="maximum">
             <number>4095</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>VOUT C</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sld_voltC">
            <property name="maximum">
             <number>4095</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spb_voltC">
            <property name="maximum">
             <number>4095</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>VOUT D</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sld_voltD">
            <property name="maximum">
             <number>4095</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spb_voltD">
            <property name="maximum">
             <number>4095</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="cbb_waveformD">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>291</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>Signal Generator</string>
      </property>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>10</y>
         <width>80</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Channel A</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Offset (0-4095)</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cbb_waveformA">
       <property name="geometry">
        <rect>
         <x>223</x>
         <y>40</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sin</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Square</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Triangle</string>
        </property>
       </item>
      </widget>
      <widget class="QDoubleSpinBox" name="dspb_freqA">
       <property name="geometry">
        <rect>
         <x>223</x>
         <y>70</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="value">
        <double>10.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>75</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Waveform</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>101</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Amplitude (0-2048)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>75</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Frequency (Hz)</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chb_generateA">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>180</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spb_offsetA">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>140</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>4095</number>
       </property>
       <property name="value">
        <number>2048</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="spb_ampA">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>110</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>2048</number>
       </property>
       <property name="value">
        <number>2048</number>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSlider">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>110</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>2048</number>
       </property>
       <property name="value">
        <number>2048</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSlider_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>140</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>4095</number>
       </property>
       <property name="value">
        <number>2048</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <zorder>label_10</zorder>
      <zorder>label_14</zorder>
      <zorder>label_10</zorder>
      <zorder>cbb_waveformA</zorder>
      <zorder>dspb_freqA</zorder>
      <zorder>label_7</zorder>
      <zorder>label_9</zorder>
      <zorder>label_8</zorder>
      <zorder>label_7</zorder>
      <zorder>label_9</zorder>
      <zorder>label_8</zorder>
      <zorder>chb_generateA</zorder>
      <zorder>spb_offsetA</zorder>
      <zorder>spb_ampA</zorder>
      <zorder>horizontalSlider</zorder>
      <zorder>horizontalSlider_2</zorder>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>327</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sld_voltA</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spb_voltA</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sld_voltB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spb_voltB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sld_voltC</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spb_voltC</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sld_voltD</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spb_voltD</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spb_voltA</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sld_voltA</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spb_voltB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sld_voltB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spb_voltC</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sld_voltC</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spb_voltD</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sld_voltD</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spb_ampA</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>470</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spb_ampA</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>478</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>481</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spb_offsetA</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>505</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>505</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spb_offsetA</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>502</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>498</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
